app/controllers/api/v2/containers_controller.rb

```diff
@@ -0,0 +1,168 @@
+# module ForemanDocker
+module Api
+  module V2
+    class ContainersController < ::Api::V2::BaseController
+      before_filter :find_resource, :except => %w(index create)
+
+      resource_description do
+        resource_id 'containers'
+        api_version 'v2'
+        api_base_url '/api/v2'
+      end
+
+      api :GET, '/containers/', N_('List all containers')
+      api :GET, '/compute_resources/:compute_resource_id/containers/',
+          N_('List all containers in a compute resource')
+      param :compute_resource_id, :identifier
```

Comment:
Nowhere actually, I should add the routes so that compute_resources picks this up.